frontend/packages/kubevirt-plugin/src/components/modals/disk-modal/disk-modal.tsx

```diff
@@ -354,12 +348,19 @@ export const DiskModal = withHandlePromise((props: DiskModalProps) => {
   return (
     <div className="modal-content">
       <ModalTitle>
-        {isEditing ? EDIT : ADD} {type.toString()}
+        {isEditing ? t('kubevirt-plugin~Edit') : t('kubevirt-plugin~Add')}{' '}
+        {t('kubevirt-plugin~{{type}}', { type: type.toString() })}
```

Comment:
Notes to next pass:
this will not get translated unless we have comments with `// t('kubevirt-plugin~<all posible types... >` ... do we need it translated ?

Reply:
You're right. I added these earlier on before I understood about the comments hack. I've been meaning to take them out and forgot. I'll remove them in the next pass.